rosdep/python.yaml

```
@@ -9296,6 +9296,19 @@ python3-suas-interop-clients-pip:
   ubuntu:
     pip:
       packages: [suas-interop-clients]
+python3-supervision-pip:
+  debian:
+    pip:
+      packages: [supervision]
+  fedora:
+    pip:
+      packages: [supervision]
+  osx:
+    pip:
+      packages: [supervision]
+  ubuntu:
+    pip:
+      packages: [supervision]
```

```suggestion
python3-supervision-pip:
  '*':
    pip:
      packages: [supervision]
```

Use platform wildcard.